.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,48 @@
-**What does this PR do / Why do we need it**:
+**What this PR does / why we need it**:
 
-**Does this PR close any issues?**:<!-- optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged -->
+**Which issue(s) this PR fixes**:
+<!--optional, in `fixes #<issue number>` format, will close the issue(s) when PR gets merged-->
 Fixes #
 
+**What type of PR is this?**
+<!--
+Add one of the following kinds:
+/kind bug
+/kind cleanup
+/kind documentation
+/kind feature
+/kind design
+
+Optionally add one or more of the following kinds if applicable:
+/kind api-change
+/kind deprecation
+/kind failing-test
+/kind flake
+/kind regression
+/kind question
+-->
+
 **Special notes for your reviewer**:
 
 **Documentation**:
-<!-- Add links to the related documentation changes related to this pull request. E.g. the link to the kubermatic/docs pull request. -->
+<!--
+Please do one of the following options:
+- Add a link to the existing documentation
+- Add a link to the kubermatic/docs pull request
+- If no documentation change is applicable then add:
+  - TBD (documentation will be added later)
+  - NONE (no documentation needed for this PR)
+-->
+```documentation
+
+```
```

Comment:
I would put this below the release-note block. I think the flow would be a little bit better.